TypeDBConsole.java

```diff
@@ -444,6 +456,10 @@ private TypeDBClient createTypeDBClient(CLIOptions options) {
                 String optCluster = options.cluster();
                 if (optCluster != null) {
                     client = TypeDB.clusterClient(set(optCluster.split(",")), createTypeDBCredential(options));
+                    Optional<Long> passwordExpiryDays = client.asCluster().users().get(options.username).passwordExpiryDays();
+                    if (passwordExpiryDays.isPresent() && passwordExpiryDays.get() <= PASSWORD_EXPIRY_WARN_DAYS) {
+                        printer.info("Your password will expire in " + passwordExpiryDays + " days.");
```

Comment:
does this need a `.get()` on passwordExpiryDays, otherwise it'll print as an `Optional[...]`?

Reply:
Nice catch, added.